tools/rdm/static/rdmtests.html

```diff
@@ -196,8 +196,6 @@
       }
 
       #rdm-tests-results-list {
-        margin-left: 20px;
-        margin-bottom: 20px;
```

Comment:
I assume these are duplicate something else, but I couldn't spot it as easily this time.

Reply:
They are overridden by `margin: 0px auto` a few lines below.